README.adoc

```diff
@@ -1,102 +1,405 @@
+TIP: For example, you are able to set standard technical diagram themes for consistency across your organization or team without any changes to the diagrams themselves.
+
+Additional{nbsp}resources::
++
+* *Theme/Stylesheet*: Default
+* *Add a stylesheet*: None (default)
+
+Javascript::
++
+* *Load the Javascript*: Before (default)
+
+== Enterprise installations
+For enterprises using good practices to providing tools and air gapped installations, the approach is to take the release artifacts, use your scanning and validation processes, and to hold them in your internal software distribution platform. 
+The guides to install from your local library can then be updated and may also reference this material.
+
+[#local-installation]
+== Local installations
+Local installations are for users and developers when:
+
+* Your machine does not have access to the internet or extension stores
+* You are testing an unreleased (locally built) version
+
+Loading your local copy of the extension on {chromium-browser-list} or Firefox is super easy!
+
+=== {chromium-browser-list}
+
+ 1. Open the `chrome://extensions` page and enable the *Developer mode* option
+ 2. Click button btn:[Load unpacked] and select the extension `app` directory of the unpacked extension files
+
+To see your changes, click the *Reload*  (kbd:[Ctrl+R]) link in `chrome://extensions` page.
+If you want to create a `Pack extension` then make a `zip` file of the extension `app` directory.
+
+=== Firefox
+
+ 1. Click the button btn:[Load Temporary Add-on] on the `about:debugging` page
+ 2. Select the manifest file in the `app` directory
 
-The options page let you add custom Asciidoctor attributes or change the theme of the AsciiDoc HTML output.
+To see your changes, click the *Reload* link in `about:debugging` page.
+
+== Known issues
+These are the currently known issues.
+For detailed issue tracking, see the GitHub {uri-project-repo}/issues[issue list].
+
+=== Known Firefox issues
+With Firefox, pages viewed directly from GitHub may not render properly: ::
+This is result of an open Firefox bug carrying over the Content Security Policies (CSP).
+The restrictive CSP sent by GitHub are applied to the content inserted by the plugin.
++
+Firefox bug tracker https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[bugzilla.mozilla.org#1267027].
+Last updated 2023-04-07 as at 2023-04-31.
+
+With Firefox, the extension does not automatically trigger when navigating to a `file://` from a link: ::
++
+You have to enter the `file://` URL directly into the address bar of the browser.
++
+If you do navigate to a `file://` URL from a link, click in the address bar and press "Enter" to force the extension to load.
+Caused by an open Firefox bug to fix this limitation.
++
+Firefox bug tracker https://bugzilla.mozilla.org/show_bug.cgi?id=1266960[bugzilla.mozilla.org#1266960].
+Last updated 2023-02-22 as at 2023-04-31].
 
 == Contributing
 
-See the <<hacking#, hacking>> file.
+See the <<CONTRIBUTING#>> file.
```

Comment:
Interdocuemnt xrefs should always have a file extension. I also think they should be written using the xref macro. I like to reserve the xref shorthand for internal references as it makes it clearer that's what they are.

This would be:

```
xref:CONTRIBUTING.adoc[contributing].
```